packages/js/manifests/polywrap/src/formats/polywrap.test/migrators/0.1.0_to_0.2.0.ts

```diff
@@ -0,0 +1,67 @@
+import { ILogger } from "@polywrap/logging-js";
+import { Jobs as OldJobs, PolywrapWorkflow as OldManifest } from "../0.1.0";
+import { Jobs as NewJobs, PolywrapWorkflow as NewManifest } from "../0.2.0";
+
+export function migrate(manifest: OldManifest, logger?: ILogger): NewManifest {
+  const [jobsSanitized, jobsHaveStepsWithConfig] = sanitizeJobsSteps(
+    manifest.jobs
+  );
+
+  if (jobsHaveStepsWithConfig) {
+    logger?.warn(
```

Comment:
This warning is great

Reply:
this warning made me think we probably want to have translations for these messages? since they're thrown through the CLI :thinking: anyways, a topic for another PR in any case :yum:

Reply:
The 1st time I issued such a warning I also thought of this. However, it's a bit hard to reason around this without extracting intl into its own package, as currently only the CLI has intl support. That's why I went this route and said "We'll cross that bridge when we get there."

We're also not using intl right now (as both `en` and `es` are in English), so I think it's a low-priority issue right now.